sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -0,0 +1,792 @@
+## API Report File for "@azure/ai-text-analytics"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { CommonClientOptions } from '@azure/core-client';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-client';
+import { TokenCredential } from '@azure/core-auth';
+
+// @public
+export interface ActionCommon {
+    disableServiceLogs?: boolean;
+}
+
+// @public
+export type ActionPrebuilt = ActionCommon & {
+    modelVersion?: string;
+};
+
+// @public
+export const AnalyzeActionName: readonly ["EntityLinking", "EntityRecognition", "KeyPhraseExtraction", "PiiEntityRecognition", "LanguageDetection", "SentimentAnalysis"];
```

Comment:
is this what the user passes into `analyze()`? if so, should we drop the `action` part?

Reply:
@kristapratico, I am not sure if I follow your concern here so forgive me if I am off the mark. The user typically passes the action name to `analyze` as shown in the demo in the PR description. `AnalyzeActionName` is just a list of all possible names and I don't think a customer would want to index that list to get an action name to be passed to `analyze`.

Reply:
Sorry this comment is really the same as https://github.com/Azure/azure-sdk-for-js/pull/21417#discussion_r853332753

I was just hoping we might be able to brainstorm some naming that might make super clear whether a user should expect "fast" results or an LRO. With using *Action with both, I don't know if it will be confusing or not. Maybe this is something we can user test.

Reply:
Good point, I was thinking that the other method will have the prefix `begin` and will return a poller so that may be enough to set their expectations about the latency.